components/omega/CMakeLists.txt

```diff
@@ -44,7 +44,7 @@ if (NOT DEFINED PROJECT_NAME)
 
   set(CMAKE_CXX_STANDARD 17) # used in E3SM
   set(CMAKE_CXX_STANDARD_REQUIRED ON)
-  set(LINKER_LANGUAGE C)     # needed to support Cray C compiler wrapper
+  set(LINKER_LANGUAGE CXX)   # needed to support Cray C compiler wrapper
```

Comment:
On Frontier, there was an issue with linking to the CXX compiler because mipcxx was set to CXX instead of the CC Cray compiler wrapper. During the linking phase, the Omega compilation failed. Time have passed since then, and a quick build with this change now seems to pass the linking stage, but please ensure that this change works correctly with all compilers on Frontier.